.github/workflows/test.yml

```diff
@@ -28,12 +28,42 @@ jobs:
       - name: Test
         run: make test
 
-      - name: Track docs
+      - name: Test if it's a release PR
```

Comment:
I would rather use an other workflow and filter on path. Instead, it forces you to use bash script tricks.
At least I would use conditional jobs instead of conditional steps.